packages/pages/src/bundler.js

```diff
@@ -36,7 +36,7 @@ const commonBuildOpts = {
   loader: {
     ".ts": "ts",
     ".html": "text",
-    ".png": "file",
+    ".png": "copy",
```

Comment:
copy is a type?

Reply:
[https://esbuild.github.io/content-types/#copy](https://esbuild.github.io/content-types/#copy)

From what I can tell, the two esbuild loaders for something like an .ico are `file` or `copy`. `file` creates a JS file with a default export being the relative path to the file. But, since `server.ts` isn't at the same directory level as the `server/public` folder, the relative path was incorrect and required string modification. 

The documentation reads like `copy` is supposed to move the image to the bundle root and then update imports. But I could only get it to keep the file where it is in the source code (and if I try to import it node throws a `Unknown file extension` error. However, since it's in the same spot as it is in the source code, I can just create a path in `server.ts` that points to the file.

Reply:
Ah ok.